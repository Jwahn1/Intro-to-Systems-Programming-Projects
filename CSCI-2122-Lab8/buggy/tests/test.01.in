new
append Alice
empty
prepend Carol
size
end
