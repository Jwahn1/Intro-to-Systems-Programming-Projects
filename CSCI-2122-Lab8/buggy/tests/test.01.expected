[ ]
[ Alice ]
List is not empty
[ Carol Alice ]
List has 2 items
